tests/CollectionTest.php

```diff
@@ -485,6 +485,55 @@ public function testReverse(): void
         self::assertEquals([5, 4, 3, 2, 1], CollectionStub::fromIterable([1, 2, 3, 4, 5])->reverse()->toArray());
     }
 
+    #[DataProvider('chunkDataProvider')]
+    public function testChunk(Collection $collectionToChunk, int $chunkSize, array $expectedChunks): void
```

Comment:
No need to be verbose. `$collection` is a valid name as we are TESTING the CHUNKING of a COLLECTION (so no ambiguity here, let's keep it simple):

```php
    #[DataProvider('chunkDataProvider')]
    public function testChunk(Collection $collection, int $chunkSize, array $expectedChunks): void
    {
        self::assertEquals($expectedChunks, $collection->chunk($chunkSize));
    }
```

Reply:
Will un-verbose it.